lib/load_tester.rb

```diff
@@ -1,4 +1,8 @@
 class LoadTester
+  def initialize
```

Comment:
Do we need this (or should `enable_logging` be uncommented?)

Reply:
It's useful to uncomment while running load tests to see where performance could be increased, that's the only reason I've left it there as we have no better way of passing in arguments to the load tester at the moment.

Reply:
:+1: